projects/components/src/confirmation/confirmation-modal.component.ts

```diff
@@ -9,7 +9,10 @@ import { ModalRef, MODAL_DATA } from '../modal/modal';
   template: `
     <div class="confirmation-modal">
       <div class="description">
-        {{ this.descriptionText }}
+        <div *ngIf="this.descriptionText; else templateRenderer" class="text">{{ this.descriptionText }}</div>
+        <ng-template #templateRenderer>
```

Comment:
I don't believe we need to rewrap this in a new template (although not positive) - can you just do `else.this.renderer` above?